internal/commands/serverstorage/attach.go

```diff
@@ -29,7 +29,12 @@ type attachParams struct {
 // AttachCommand creates the "server storage attach" command
 func AttachCommand() commands.Command {
 	return &attachCommand{
-		BaseCommand: commands.New("attach", "Attach a storage as a device to a server", ""),
+		BaseCommand: commands.New(
+			"attach",
+			"Attach a storage as a device to a server",
+			"upctl server storage attach 00038afc-d526-4148-af0e-d2f1eeaded9b --storage 015899e0-0a68-4949-85bb-261a99de5fdd",
+			`upctl server storage attach my_server1 --storage "My Storage"`,
```

Comment:
maybe could add examples of --address use here as well? I had problems understanding it to begin with. The flag description has since been clarified but might make sense to add examples as well?

Reply:
done :)